WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.java

```diff
@@ -139,4 +141,18 @@ public static void goToHomeTab() {
     public static void goToMenuTab() {
         selectItemWithTitleInTabLayout(getTranslatedString(R.string.my_site_menu_tab_title), R.id.tab_layout);
     }
+
+    /**
+     * Clicks on the "Quick Action" item or the Site menu item if the quick actions card is hidden.
+     * Needed because locating site menu items by text fails if the quick actions are available.
+     * @param quicActionItemId Id of the quick actions menu item.
+     * @param siteMenuItemString String resource id of the site menu item.
+     */
+    private void clickQuickActionOrSiteMenuItem(@IdRes int quicActionItemId, @StringRes int siteMenuItemString) {
```

Comment:
@ovitrif - great idea on this method. I think there _could_ be an experiment on quick actions + it might work different for WP vs. JP. 🤔  experiments and tests are not a good combination.

Reply:
Thank you 🙇 

> experiments and tests are not a good combination
Got to the same conclusion 😅